TAO/tests/Bug_3940_Regression/README

```diff
@@ -0,0 +1,16 @@
+
+
+This is a compile only test. If it compiles, it's passed.
```

Comment:
Since it's a compile-only test, it shouldn't need a run_test.pl or a test.cpp (build a library instead of an executable).  Or even use `custom_only=1` and just run tao_idl without compiling C++ code.

Reply:
Thanks, wilco.